renovate.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": [
+    "config:base",
+    "schedule:weekly"
```

Comment:
Per, https://docs.renovatebot.com/presets-schedule/; what about something like:

```js
{
  ...,
  "extends": [
    "schedule:earlyMondays"
  ]
}
```

Might just mean 1 day of PR pain where we just triage everything on Monday morning and go merge crazy and hope it isnt conflict hell.